### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by normandy7**
- Remove outdated elements
- Update and reformat code samples
- Add version warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,80 @@
...
 
 In your Jupyter Notebook environment, some Neptune items appear in your toolbar.
-1. Click the Neptune icon to connect with Neptune.
-2. Click **Upload** to create a first notebook checkpoint in Neptune.
-3. In the Jupyter notebook, import Neptune and start a run:
+
+1. To connect with Neptune, click the Neptune **Configure** button.
+2. Enter your [Neptune credentials](https://docs.neptune.ai/setup/setting_credentials/#finding-your-credentials).
+3. To register the notebook in Neptune, click **Upload**.
+4. To log metadata from the notebook automatically, in the top navigation menu, click **Activate**.
```
Comment (SiddhantSadangi) [inline README.md:45]: Activating doesn't automatically log metadata, it just links the runs created in the notebook to the notebook
image


```suggestion
4. To link runs to the notebook automatically, in the top navigation menu, click **Activate**.
```

**Review by SiddhantSadangi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (normandy7) [reply README.md:45]: Ahh, but... Wait a minute. What does that _do_, exactly? We used to have this "Used by X runs" for notebooks, right? Is it about that, or something else?

Comment (SiddhantSadangi) [reply README.md:45]: ah yes, we removed the "Used by X runs" feature.

I don't think "activating" does anything then. Uploading the initial checkpoint to Neptune takes care of logging the notebook in `source_code/notebook`.

Lemme confirm though

**Review by Patrycja-J — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,75 +1,81 @@
...
+```
 
-# Stop the run
-run.stop()
+When you're done with the logging, stop the run:
 
-# See notebook snapshots in Neptune under "Source code"
+```python
+run.stop()
 ```
```
Comment (SiddhantSadangi) [inline README.md:71]: Let's mention where they can find the uploaded notebook - in the "Source Code" namespace, and in the "Notebooks" tab in project.

**Review by SiddhantSadangi — CHANGES_REQUESTED** [review]

**Review by SiddhantSadangi — APPROVED** [review]

**Pull request merged**